src/main/java/org/testeditor/fixture/host/HostDriverFixture.aml

```diff
@@ -0,0 +1,16 @@
+package org.testeditor.fixture.host
+
+interaction type connect {
+    template = "Connect to " ${hostname} " with path " s3270Path " and port " ${port}  
```

Comment:
The spaces in the template are not necessary. Please write `"Connect to" ${hostname} "with path"`